.github/templates/common.yml.j2

```diff
@@ -7,8 +7,8 @@
 
 # NOTE: If testing pytorch/builder changes you can change this variable to change what pytorch/builder reference
 #       the binary builds will check out
-{%- set builder_repo = "pytorch/builder" -%}
-{%- set builder_branch = "main" -%}
+{%- set builder_repo = "tinglvv/builder" -%}
+{%- set builder_branch = "126-windows-change" -%}
```

Comment:
Hi @tinglvv you probably want to put in back to main so that changes in this PR https://github.com/pytorch/builder/pull/2025 can become visible